range_iterator.py

```diff
@@ -0,0 +1,23 @@
+class MyRange:
```

Comment:
I would be better to create a different name for this one.

Reply:
done